bakery/checkers/namespace.go

```diff
@@ -175,6 +174,9 @@ func invalidSchemaRune(r rune) bool {
 	return unicode.IsSpace(r)
 }
 
+// IsValidSchemaURI reports whether the given URI is valid for use as
+// a namespace schema URL. It must be non-empty,
```

Comment:
What's the distinction between URI and URL here, should the function be called IsValidSchemaURL?